package.json

```diff
@@ -4,19 +4,40 @@
   "private": true,
   "scripts": {
     "start": "node node_modules/react-native/local-cli/cli.js start",
-    "test": "jest"
+    "eslint": "eslint src/**",
+    "test": "jest --forceExit"
   },
   "dependencies": {
+    "antd-mobile": "^2.1.3",
+    "axios": "^0.17.1",
     "react": "16.2.0",
-    "react-native": "0.52.0"
+    "react-native": "0.52.0",
+    "react-navigation": "^1.0.0-beta.27",
+    "react-redux": "^5.0.6",
+    "redux": "^3.7.2",
+    "redux-persist": "^5.5.0",
+    "remote-redux-devtools": "^0.5.12"
```

Comment:
it could be in devDependencies

Reply:
Yes, however ESLint throws `'remote-redux-devtools' should be listed in the project's dependencies, not devDependencies` so I'm keeping it there.